flags.yml

```diff
@@ -79,6 +79,14 @@
   lifetime: temporary
   expose: true
 
+- name: Notebooks Service API
```

Comment:
Are we creating a 2nd flag (vs. reusing / renaming the existing one) to support both old and new versions of the UI? Or for some other reason?

Reply:
We'll have 2 flags for now to support the newest versions of notebooks from the UI. The `notebooks` flag makes the UI show the notebooks link on the sidebar, and the `notebooksApi` makes it use the "new" API. The "new" API will become the only API very soon, so for us at this point I don't think there's any reason to support the old API. I also expect that the `notebooksApi` flag will go away soon and it will be just back to the one `notebooks` flag to enable the feature with the "new" (and only) API.